scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-6a.yaml

```diff
@@ -0,0 +1,198 @@
+---
+# 6a - 7 nodes: HA IMPI 1 x 3, KVM x 4
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: ipmi_barclamp
+      sbd:
+        nodes:
+          "@@controller1@@":
+            devices:
+            - ''
+          "@@controller2@@":
+            devices:
+            - ''
+          "@@controller3@@":
+            devices:
+            - ''
+      per_node:
+        nodes:
+          "@@controller1@@":
+            params: ''
+          "@@controller2@@":
+            params: ''
+          "@@controller3@@":
+            params: ''
+    drbd:
+  deployment:
+    elements:
+      pacemaker-cluster-member:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+      hawk-server:
+      - "@@controller1@@"
+      - "@@controller2@@"
+      - "@@controller3@@"
+
+- barclamp: nfs_client
+  name: glance
+  attributes:
+    exports:
+      images:
+        nfs_server: 10.162.26.129
```

Comment:
Wrong IP, the job is preparing nfs shares on 10.162.24.1, see here: https://github.com/SUSE-Cloud/automation/pull/1181/files#diff-56165da18ec5aff63391672593647341R139